torchtext/transforms.py

```diff
@@ -657,6 +659,41 @@ def __prepare_scriptable__(self):
         return self
 
 
+class RegexTokenizer(Module):
+    __jit_unused_properties__ = ["is_jitable"]
+    r"""Regex tokenizer for a string sentence that applies all regex replacements defined in patterns_list.
+
```

Comment:
Let's update the docstring with the example we had in the `regex_tokenizer` factory method to show our users how to use this tokenizer